BotRoyaleUnity/Assets/Scripts/PartHandler.cs

```diff
@@ -9,13 +12,17 @@ public class PartHandler : MonoBehaviour
 
     public int[] directionStrength = new int[4];
 
+    private Vector2Int centerPos;
+
 
     //array to calculate all attached parts.
 
     public bool[,] attachedParts;
 
     private Rigidbody rb;
 
+    private SocketIOController socket;
```

Comment:
We shouldnt need to use the Socket.IO library's classes directly, we have the SocketConnectionHandler class to wrap the library